src/istgt_lu_disk.c

```diff
@@ -269,9 +270,24 @@ find_first_bit(ISTGT_LU_CMD_Ptr lu_cmd)
 				:"=a" (res), "=&c" (d0), "=&D" (d1)
 				:"0" (0ULL), "1" (nbits), "2" (addr),
 				[addr] "r" (addr) : "memory");
-		res_final +=res;
-		if((unsigned long)res != nbits)
+		res_final += res;
+		if ((unsigned long)res != nbits)
 			return res_final;
+#elif defined(__aarch64__)
+		unsigned long idx;
+		res = 0;
+		for (idx = 0; idx < nbits; idx++) {
+			if (addr[idx]) {
+				res += __builtin_ffsl(addr[idx]) - 1;
+				break;
+			} else {
+				res += 64;
```

Comment:
use sizeof (addr[0]) or sizeof (*addr) instead of contant.

Reply:
yeah, thanks @pawanpraka1 . These changes are great. I will update them.

Reply:
One question is, the value of sizeof(*addr) is 8, but we should plus 64 here.

Reply:
Should I use `sizeof(*addr) * 8`or other methods ?

Reply:
good point. sizeof returns bytes, we should convert that to bits here.

Reply:
we can do sizeof(*addr) << 3 and put some comments saying we need bits.

Reply:
OK, I will update it now ^^